Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -90,6 +123,16 @@ const SubmissionDetails = () => {
                   type={HeadingType.secondary}
                   className={styles.taskHeading}
                 >
+                    <span className={styles.lessThan}>
+                        &lt;
```

Comment:
Isn't there an icon for this?

Reply:
There was an icon for right arrow, so I made one for left arrow as well and am using it.